### ISSUE TITLE
Zmq Unknown error (110) from router socket

### ISSUE THREAD

**Issue by gillima** [zeromq/clrzmq4#30]
I use a router socket with `router.RouterMandatory = RouterMandatory.Report`

According to http://api.zeromq.org/4-0:zmq-setsockopt#toc24 the router should return EHOSTUNREACH when a message can not be routed but i get `Unknown error (110)` when i close the connected dealer socket.

``` csharp
try
{
   this.router.SendMore(new ZFrame(client.Id));
   this.router.Send(new ZFrame(rawMessage));
}
catch (ZException ex)
{
   System.Diagnostics.Debug.Assert(ex.ErrNo == ZError.EHOSTUNREACH);
}
```

**Comment by metadings** [comment]
Well on Linux (Posix) systems there is ETIMEDOUT == 110... but I don't know...?

**Comment by sigiesec** [comment]
This is probably fixed by #98. The error codes were not correct on Windows.
